aws-rds-cfn-common/src/main/java/software/amazon/rds/common/handler/Tagging.java

```diff
@@ -30,21 +30,21 @@
 import software.amazon.rds.common.error.ErrorStatus;
 
 public final class Tagging {
-    public static final ErrorRuleSet SOFT_FAIL_IN_PROGRESS_TAGGING_ERROR_RULE_SET = ErrorRuleSet
+    public static final ErrorRuleSet IGNORE_LIST_TAGS_PERMISSION_DENIED_ERROR_RULE_SET = ErrorRuleSet
```

Comment:
Is it ListTagsForResource permission missing from the call or is it AddTagsForResource?

Reply:
This is only used in read handlers for ListTagsForResource. Unfortunately, the RDS APIs for describing are not consistent. The DescribeDBInstance and DescribeDBCluster APIs return tags in the response, but other APIs like DescibeDBClusterEndpoint and DescribeDBParameterGroup do not. Instead, we need to make a separate call to ListTagsForResource to retrieve the tags. If this call fails due to permission denied, we ignore it to prevent the entire read handler operation from failing. As a result, the tags will not be included because the customer does not have permission to see them. We will remove this workaround once the Describe* RDS APIs are consistent and all of them return tags in the Describe* response.